pages/index.tsx

```diff
@@ -87,13 +92,73 @@ const HeroSection = () => {
         </Stack>
       </VStack>
     </Flex>
+    // </Flex>
+  )
+}
+
+const MobileHeroSection = () => {
+  return (
+    <Flex m = '25px' justifyContent={'center'}>
+      <Box borderRadius='10px' overflow='hidden' bgColor={'white'}>
+        <Image
+          alt='Mountains'
+          src={HeroBg}
+          placeholder='blur'
+          quality={100}
+          sizes='100vw'
+        />
+
+        <Box p='6'>
+          <Flex justifyContent={'center'}>
+            <Text 
+              as="span"
+              color={'#f90'}
+              fontFamily={'Domine'}
+              fontWeight='bold'
+              lineHeight={0.8}
+              fontSize='2em'
+            >
+              Cinnamon&nbsp;
+            </Text>
+            <Text 
+              as="span"
+              color={'black'}
+              fontFamily={'Domine'}
+              fontWeight='bold'
+              lineHeight={0.8}
+              fontSize='2em'
+            >
+              Student Life
+            </Text>
+          </Flex>
+        </Box>
+
+      </Box>
+    </Flex>
   )
 }
 
 const LandingPage: NextPage = () => {
   const landingPageData = landingPageMockData
-  return (
-    <Flex justify='center' flexDir='column' as='main' gap='0'>
+  const [isMobile] = useMediaQuery("(max-width: 768px)")
+
+  if (isMobile) {
+    return(
+      <Flex bgColor="#f5f5f5" justify='center' flexDir='column' as='main' gap='0'>
+        <MobileHeroSection />
+
+        <Box>
+          {landingPageData.map((data, index) => (
+            <MobileLandingPageBanner key={index} left={index % 2 == 1} {...data} />
+          ))}
+        </Box>
+
+        <Footer />
+      </Flex>
+    )
+  } else {
+    return(
```

Comment:
Style tip! There's no need to add the `else` clause for this. It makes the code less nested and easier to read.